calico_node/Makefile

```diff
@@ -498,6 +500,9 @@ node-test-containerized: vendor run-etcd-host run-k8s-apiserver
 	-v $(VERSIONS_FILE):/versions.yaml:ro \
 	-e VERSIONS_FILE=/versions.yaml \
 	-e LOCAL_USER_ID=$(LOCAL_USER_ID) \
+	-e DATASTORE_TYPE=$(DATASTORE_TYPE) \
+	-e ETCD_ENDPOINTS=http://$(LOCAL_IP_ENV):2379 \
+	-e KUBECONFIG=$(KUBECONFIG_PATH) \
```

Comment:
I might be misreading this, but is this path correct?  This needs to be the path inside the container IIUC and it doesn't look like we are volume mounting that in.